clode/python/runtime.py

```diff
@@ -1,18 +1,71 @@
 import os
 
-from . import clode_cpp_wrapper as _clode
+from . import clode_cpp_wrapper as _clode  # type: ignore
+from .clode_cpp_wrapper import log_level, print_opencl, query_opencl  # type: ignore
 
 _runtime = None
```

Comment:
Instead of having a singular global _runtime, it would be nice to for CLODEFeatures and CLODETrajectory to own a reference to their own OpenCLResource.  That way, you can have features running on a GPU while trajectory runs on the CPU.